charts/argo-cd/templates/argocd-configs/argocd-notifications-cm.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "argo-cd.notifications.configMapName" . }}
+  name: argocd-notifications-cm
```

Comment:
Why hardcoding CM name?

Reply:
Aligning with upstream
https://github.com/argoproj/argo-cd/blob/master/manifests/base/notification/argocd-notifications-cm.yaml

Reply:
Plus the hardcoded names in Argo CLI